src/const.ts

```diff
@@ -32,6 +32,9 @@ export const BATCHES_TO_WAIT = 3
 // Furtherst batch id possible (uint32), must be a js Number
 export const MAX_BATCH_ID = 2 ** 32 - 1
 
+// How many orders should we query per call, when invoking https://github.com/gnosis/dex-contracts/blob/master/contracts/BatchExchange.sol#L479
+export const DEFAULT_ORDERS_PAGE_SIZE = 10
```

Comment:
Why 10. I find it a bit small. We should maybe see a good number that return the maximum results but is still performant.

Reply:
Arbitrary number, no particular reason. What do you suggest? 20? 50? 100?

Reply:
Yes, I would try to see how it behaves with a bigger number
Maybe we can do this when we do the part of fetching all the pages (finding the right number that is a good balance for batch fetching)

Reply:
This is what I'm considering. I'll adjust the number based on the next PRs where I actually use this method.
Will merge like it is for now.